tools/shared.py

```diff
@@ -320,10 +320,14 @@ def fix_js_engine(old, new):
 
 # Expectations
 
-EXPECTED_LLVM_VERSION = (4, 0)
-
 actual_clang_version = None
 
+def expected_llvm_version():
+  if get_llvm_target() == WASM_TARGET:
```

Comment:
Hm. I agree that it would probably work today with the version in fastcomp. Although the test expectations at least assume that we have tip-of-tree wasm backend. And when we add something new that breaks compatibility, we probably won't remember to come back and update this expectation. I'd lean toward doing the warning as written here, indicating that this is the version we test and expect to work. And it is just a warning, that the user can ignore if they want.